src/components/booksearch/BookSearch.jsx

```diff
@@ -0,0 +1,65 @@
+import './booksearch.css';
+import books from '../../data/books.json';
+
+const BookSearch = () => {
+  return (
+   
+   
+   
+   
+   <main  className="booksearch__backgroundandscroll">
+
+
+
+
+
+
+
+    <div className="booksearch">
+
+      <div >
+        <h1>logo&apos;s space</h1>
+      </div>
+      <div className="booksearch__searcher">
```

Comment:
```suggestion
      <div className="booksearch__search">
```
I think `search` is descriptive enough